Lib/test/libregrtest/runtest_mp.py

```diff
@@ -66,7 +66,9 @@ def run_test_in_subprocess(testname: str, ns: Namespace, tmp_dir: str, stdout_fh
            '-m', 'test.regrtest',
            '--worker-args', worker_args]
 
+
     env = dict(os.environ)
+    env['PYTHONIOENCODING'] = 'utf-8:surrogateescape'
```

Comment:
I would prefer to test the Python "default" encoding, and so fix code which decodes test stdout instead.

Reply:
Python test suite worked well with any encoding for years without having to force an encoding, it was broken recently to support WASM.